RFC_PROCESS.md

```diff
@@ -0,0 +1,39 @@
+# RFC Process
+
+## What's an RFC?
+
+The name itself is a reference to the IETF's Request For Comments process, and basically involves a document or series of documents which are drafted, reviewed, and eventually ratified (approved) by the Ayo.js community through discussion among those interested.
+
+An RFC can extend, modify, or alter any part of the Ayo.js project's code or documents, whether or not they've been previously documented. This includes proposing new features, proposing major breaking changes, changes to the Code of Conduct and other community policies, changes to the governance structure of the project, and anything else that affects a significant part of the community. It can also propose entirely new policies and community agreements.
```

Comment:
`code or documents` – I’m not sure, do you have anything specific in mind when you say that? Proposing API changes or sth like that?

(also: mind if I push here with 80 character line wrapping again? I’m sorry if that’s annoying but if nothing else, it’s nicer for many editors + makes pointing to specific parts of the doc easier)

Reply:
Yep. Definitely things like proposing APIs. Or, for example, things like WebWorkers, changes to .mjs handling, etc etc.